src/handlers/itemStore.ts

```diff
@@ -121,5 +121,90 @@ export const itemStoreHandlers: NatsHandler[] = [
                 }
             }
         }
-    ]
+    ],
+    [
+        "item_store_inventory_add_item",
+        async (subscription: Subscription): Promise<void> => {
+            const client = getClient();
+
+            for await (const message of subscription) {
+                const inventory_item = jsonCodec.decode(message.data) as JSONInventoryItem;
+
+                try {
+                    const reponse = await client.index({
+                        index: ELASTICSEARCH_INVENTORY_INDEX_NAME, 
+                        id: String(inventory_item.user_id),
+                        body: {
+                            doc : inventory_item.instance_number
+                        }
+                    });
+
+                    console.log(
+                        `[INVENTORY] Item added to ${ELASTICSEARCH_INDEX_NAME}`,
+                        reponse
+                    );
+
+                    message.respond(
+                        jsonCodec.encode({
+                            inventory_item
+                        })
+                    );
+                } catch (err) {
+                    console.error(
+                        `[INVENTORY] Error adding item to ${ELASTICSEARCH_INVENTORY_INDEX_NAME}`,
+                        err
+                    );
+
+                    message.respond(
+                        jsonCodec.encode({
+                            error: err
+                        })
+                    );
+                }
+            }
+        }
+    ],
+    [
+        "item-store_inventory_update_item",
+        async (subscription: Subscription): Promise<void> => {
+            const client = getClient();
+
+            for await (const message of subscription) {
+                const data = jsonCodec.decode(message.data) as JSONInventoryItem;
+
+                try {
+                    const response = await client.update({
+                        index: ELASTICSEARCH_INVENTORY_INDEX_NAME,
+                        id: String(data.user_id),
```

Comment:
to update an inventory update we should use `client.updateByQuery` and search for the item with the combination of:

user_id + item_id + instance_number: https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/update_by_query_examples.html

the users of the platform will never know what the id for that item is in our database, it's not a concern for them, but they will know which item needs to be updated by saying: please update the item with instance_number X for item_d Y that belongs to user Z